plone/recipe/codeanalysis/entry_points.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+from tempfile import mkdtemp
+
+import os
+import subprocess
+import sys
+
+
+def check(data):
+    """Run p.r.codeanalysis when releasing with zest.releaser."""
+    from zest.releaser.utils import ask
+    if not ask('Run code analysis before tagging?'):
```

Comment:
I tried and no, it's automatically handled by zest.releaser.

Whenever it founds an ``ask`` call it replies automatically with the default answer.